app/build.gradle

```diff
@@ -36,6 +33,21 @@ android {
     }
     kotlinOptions {
         jvmTarget = '1.8'
+
+        /*********************************************************************************************************
+         | Duck tape level compile fix
```

Comment:
Supposedly this fix isn't needed in Kotlin version 1.6 i think. ( Found in one of the resources )

Reply:
Resources are a part of this comment in the code. Can't be seen on this small preview.